web/themes/joinup/templates/rdf_entity/rdf-entity--solution--group-header.html.twig

```diff
@@ -19,6 +19,9 @@
         <div class="stats__row">
           {{ content.downloads }}
         </div>
+        <div class="stats__row">
+          {{ content.field_policy_domain }}
+        </div>
```

Comment:
Off topic: I hate this. This is a wrong way to build blocks/pages. We should rely on entity displays and get some help from a Display Suite, Layout Builder or other tool. We should not add content pieces in this way.

Reply:
This was a temporary hack.. To be honest, also for me, It takes me quite some time to understand why some things are not visible in the page even though they exist in the entity display. I have created a follow up ticket for the FE design, you can request the change there. Please, disregard the FE part of the ticket. It already does not show according to the designs.